GHE_main.cpp

```diff
@@ -0,0 +1,237 @@
+//
+//  GHE_main.cpp
+//  
+//
+//  Created by Ryan Davies on 6/14/22.
+//
```

Comment:
I know most IDEs will add these headers automatically, but it can be deleted, we know what the filename is and we can track authorship through Git.